src/file-upload/file-item.tsx

```diff
@@ -124,7 +132,7 @@ export const FileItem = ({
                         >
                             <PencilIcon />
                         </IconButton>
-                    )} */}
+                    )}
```

Comment:
should set the usual `aria-label` and `aria-hidden` attributes for the edit button